contracts/src/v0.7/ChainlinkClient.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.7.0;
 import "./Chainlink.sol";
 import "./interfaces/ENSInterface.sol";
```

Comment:
Can we upgrade the contracts changelog with this change?

Reply:
Noted.